packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -191,11 +191,29 @@ export interface TldrawEditorBaseProps {
 	/**
 	 * Predicate for whether or not a shape should be hidden.
 	 *
+	 * @deprecated Use {@link TldrawEditorBaseProps#getShapeVisibility} instead.
+	 */
+	isShapeHidden?(shape: TLShape, editor: Editor): boolean
+
+	/**
+	 * Provides a way to hide shapes.
```

Comment:
nit - fairly self explanatory, but it could be worth listing and explaining each possible return value here